build.gradle

```diff
@@ -20,7 +20,7 @@ buildscript {
 }
 
 group = 'com.synopsys.integration'
-version = '8.11.2-SIGQA1'
+version = '8.11.2-SIGQA3-shanty.testing-docs-build-gradle-SNAPSHOT'
```

Comment:
@shantyk This can just be 8.11.2-SIGQA1 I think

Reply:
Ok thank you, wasn't sure if the proper suffix was -SNAPSHOT or -SIGQA#. Will do the same for the 9.10.z MR

Reply:
Reverted the version string to be 8.11.2-SIGQA1